### DISCUSSION TITLE
Defining `gridTemplateColumns` in tailwind.config.js is breaking builtin grid-cols-n utilities.

### DISCUSSION THREAD

**Discussion by Christopher-Stevers** [tailwindlabs/tailwindcss#10297] [Help] [answered]
Once I set custom values for gritTemplateColumns in the tailwind config tailwind stops generating css for the default grid-cols-_n_ utilities.
For example:
When my markup is:
```html

<div className='grid  grid-rows-2 grid-cols-2 justify-between w-full'>
                  <div>
                    <label className=' block my-2' htmlFor={'proAccount'}>
                      Topics
                    </label>
                    <input className='input-field  ' id={'proAccount'} />
                  </div>

                  <div>
                    <label className=' block my-2' htmlFor={'proAccount'}>
                      Topics
                    </label>
                    <input className='input-field ' id={'proAccount'} />
                  </div>
                </div>
   
```
When my tailwind config looks like this:
```js
               ...
		gridTemplateColumns: {
			'wide': '1fr 950px 1fr',
			'extra-wide': '.2fr 4fr .2fr',
			'annoying': 'repeat( auto-fit, 192px)',
		},
                ...
```
My browser shows:
![image](https://user-images.githubusercontent.com/72156679/211814553-f05f3467-4050-4172-85be-9b9cae5b7a46.png)
And no value is set for grid-template-columns in the devtools computed tab:
![image](https://user-images.githubusercontent.com/72156679/211815330-124d5aba-44fb-40cc-9bf3-6aec537506ca.png)

When I don't set a value for gridTemplateColumns the same markup displays as expected:
![image](https://user-images.githubusercontent.com/72156679/211815026-860a9f9e-c384-4a4f-a28b-7d4d0c93946f.png)
I'm using tailwind within Next.js
I checked out https://github.com/tailwindlabs/tailwindcss/discussions/5551 but the workaround suggested didn't work. For now I can work around it by just defining my grid-cols values in gridTemplateColumns, but it seems like this might be a bug.

**Comment by adamwathan** [comment, marked as answer]
Hey! Make sure you are defining your values under `extend` if you want to extend the defaults instead of overriding them.

https://tailwindcss.com/docs/theme#extending-the-default-theme

**Reply by Christopher-Stevers** [reply]
Thanks! Apologies for my not checking the docs well enough 🙈
